src/main/java/com/exadel/sandbox/entities/Office.java

```diff
@@ -0,0 +1,33 @@
+package com.exadel.sandbox.entities;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class Office {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private Long parkingId;
```

Comment:
Take a look at JPA relations. I think you can use OneToOne here. I don't know if it's neseccary. But I think it will be great if you use them.